docs/core/compatibility/containers/9.0/noble-arm32-images.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Breaking change: `9.0-noble` images are 64-bit only"
+description: Learn about the breaking change in Ubuntu 24.04 "noble" container images.
+ms.date: 04/29/2024
+ms.custom: linux-related-content
+---
+# `9.0-noble` images are 64-bit only
+
+.NET 9 container images for Ubuntu 24.04 are provided for Arm64 and x64 only. .NET 9 is not supported on Ubuntu 24.04 Arm32 due to [year 2038 incompatibility](../../core-libraries/9.0/y2038-incompatible.md).
+
+## Previous behavior
+
+Previously, you could reference a tag like `8.0` and be able to retrieve an Arm32 container image.
+
+## New behavior
+
+Starting in .NET 9, the `9.0` tag only retrieves 64-bit images.
```

Comment:
The [discussion here](https://github.com/dotnet/core/discussions/9285#discussioncomment-9269485) seems to indicate that we would have Arm32 support for Debian 12 which would be provided by the `9.0` tag.